galaxy_api/api/ui/viewsets/namespace.py

```diff
@@ -49,9 +51,18 @@ class NamespaceViewSet(
 
     filterset_class = NamespaceFilter
 
+    def create(self, request, *args, **kwargs):
+        serializer = serializers.NamespaceSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
```

Comment:
If `serializer.is_valid` raises Exception checking it's return value doesn't make any sense.